invenio_app_rdm/theme/assets/js/invenio_app_rdm/search/SearchResults/components/ResultsListItem/index.js

```diff
@@ -0,0 +1 @@
+export { ResultsListItem } from "./ResultsListItem";
```

Comment:
Missing copyright. Ok in many files (mostly index.js files)  hehe just check they all have it :)